modules/gitops-repo-server-properties.adoc

```diff
@@ -22,8 +22,10 @@ The following properties are available for configuring the Repo server component
 |`logFormat` | `text` |The log format to be used by the Argo CD Repo server. Valid options are `text` or `json`.
```

Comment:
Again, I am not sure about the capitalization here: "Argo CD Repo server"

Reply:
Fixed.